### PR TITLE
MOSIP-22332 adding access logs to idgen

### PR DESCRIPTION
**Pull request by urviljoshi**

### CODE REVIEW

File: kernel/kernel-idgenerator-service/src/main/resources/logback.xml [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
-	<logger name="io.vertx.core.impl.BlockedThreadChecker" level="OFF" />
+	<logger name="io.vertx.core.impl.BlockedThreadChecker"
+		level="OFF" />
+
+
+	<logger
+		name="io.mosip.kernel.idgenerator.config.AccessLogHandler" additivity="false">
+		<appender-ref ref="ACCESS" />
+	</logger>
+	l
```
Comment (vishwa-vyom) [inline kernel/kernel-idgenerator-service/src/main/resources/logback.xml]: what is this l ? And why do we need these logback changes ?

File: kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/verticle/HttpServerVerticle.java [hunk 2]
```diff
@@ -100,4 +105,17 @@ public void start(Future<Void> future) {
 			}
 		});
 	}
+
+	private void accessLogHandler(final RoutingContext context, AccessLogHandler accessLogHandler) {
```
Comment (vishwa-vyom) [inline kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/verticle/HttpServerVerticle.java]: Rename this method to addAccessLogHandler and remove unwanted line brakes inside the method

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: kernel/kernel-idgenerator-service/src/main/resources/logback.xml (hunk 1 above)
Comment (urviljoshi) [reply kernel/kernel-idgenerator-service/src/main/resources/logback.xml]: This is to remove the default pattern which is used for other logs as our json log was printed as a msg in that.

File: kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/verticle/HttpServerVerticle.java (hunk 2 above)
Comment (urviljoshi) [reply kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/verticle/HttpServerVerticle.java]: done

File: kernel/kernel-idgenerator-service/src/main/resources/logback.xml (hunk 1 above)
Comment (vishwa-vyom) [reply kernel/kernel-idgenerator-service/src/main/resources/logback.xml]: As discussed please remove the I and test it once

**Pull request merged**